ts/themes/globals.tsx

```diff
@@ -7,6 +7,10 @@ function setDuration(duration: number | string) {
   return `${!isTestIntegration() ? duration : typeof duration === 'string' ? '0s' : '0'}`;
 }
 
+export function pxValueToNumber(value: string) {
+  return Number(value.replace('px', ''));
```

Comment:
should we throw on Nan, or just non finite values?

Reply:
Good point! Fixed with NaN check.